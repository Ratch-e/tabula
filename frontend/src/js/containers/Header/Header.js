import * as React from "react";

class Header extends React.Component {
  render() {
    return (
      <div className="header">
        <div className="logo">Tabula</div>
      </div>
    );
  }
}

export default Header;