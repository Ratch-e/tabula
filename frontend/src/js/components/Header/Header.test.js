import React from 'react';
import Header from './Header.js';
import renderer from 'react-test-renderer';

describe('Шапка', () => {

  it('Рендерится корректно', () => {
    const tree = renderer.create(<Header/>).toJSON();
    expect(tree).toMatchSnapshot();
  })

});
