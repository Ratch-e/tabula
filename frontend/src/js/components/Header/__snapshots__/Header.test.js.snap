// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Шапка Рендерится корректно 1`] = `
<div
  className="header"
>
  <a
    className="logo"
    href="/"
  >
    Tabula
  </a>
</div>
`;
